auth/auth.rb

```diff
@@ -0,0 +1,67 @@
+# Copyright 2017 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def implicit project_id:
+  # [START implicit]
+  # project_id = "Your Google Cloud project ID"
+
+  require "google/cloud/storage"
+
+  # If you don't specify credentials when constructing the client, the client
+  # library will look for credentials in the environment.
```

Comment:
**User:** *"look for credentials in the environment"? What does that mean?*

It's a little vague. Is that the same wording that the other samples use? If not, maybe see if they have better language. It's very specific, but I think it would actually be very beneficial to users to specify *exactly* where we look for credentials, eg. say the library will look for credentials from the X environment variable? If there's a clear way to describe the specifics, I would *love* to know the exact environment variable / etc as a user. I mean... that's why I'm on the page that described the Authentication details in the first place! 😄